cmd/kots/cli/get-versions.go

```diff
@@ -0,0 +1,154 @@
+package cli
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+
+	"github.com/pkg/errors"
+	"github.com/replicatedhq/kots/pkg/auth"
+	"github.com/replicatedhq/kots/pkg/handlers"
+	"github.com/replicatedhq/kots/pkg/k8sutil"
+	"github.com/replicatedhq/kots/pkg/logger"
+	"github.com/replicatedhq/kots/pkg/print"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+func GetVersionsCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:           "versions [appSlug]",
+		Short:         "Get App Versions",
+		Long:          "",
+		SilenceUsage:  false,
+		SilenceErrors: false,
+		PreRun: func(cmd *cobra.Command, args []string) {
+			viper.BindPFlags(cmd.Flags())
+		},
+		RunE: getVersionsCmd,
+	}
+
+	cmd.Flags().StringP("output", "o", "", "output format (currently supported: json)")
+
+	return cmd
+}
+
+func getVersionsCmd(cmd *cobra.Command, args []string) error {
+	v := viper.GetViper()
+
+	if len(args) == 0 {
+		cmd.Help()
+		os.Exit(1)
+	}
+
+	appSlug := args[0]
```

Comment:
i thought this was a flag? at least that's what the pr description show

Reply:
The docs align with not needing a flag here, and I've updated the PR description to match the code.